openapi/index_openapi.json

```diff
@@ -1081,10 +1081,21 @@
             "description": "JSON API links object, pointing to a homepage URL for this implementation"
           },
           "link_type": {
-            "$ref": "#/components/schemas/LinkType"
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/LinkType"
+              }
+            ],
+            "description": "The type of the linked relation.\nMUST be one of these values: 'child', 'root', 'external', 'providers'."
           },
           "aggregate": {
-            "$ref": "#/components/schemas/Aggregate"
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/Aggregate"
+              }
+            ],
+            "description": "A string indicating whether a client that is following links to aggregate results from different OPTIMADE implementations should follow this link or not.\nThis flag SHOULD NOT be indicated for links where `link_type` is not `child`.\n\nIf not specified, clients MAY assume that the value is `ok`.\nIf specified, and the value is anything different than `ok`, the client MUST assume that the server is suggesting not to follow the link during aggregation by default (also if the value is not among the known ones, in case a future specification adds new accepted values).\n\nSpecific values indicate the reason why the server is providing the suggestion.\nA client MAY follow the link anyway if it has reason to do so (e.g., if the client is looking for all test databases, it MAY follow the links marked with `aggregate`=`test`).\n\nIf specified, it MUST be one of the values listed in section Link Aggregate Options.",
+            "default": "ok"
```

Comment:
Specification of a default value here is new. This makes me lean towards supporting the releasing OPTIMADE v1.0.1 after having this PR merged.

Reply:
True, it is, for this field. It's however reflecting the current specification, which states that this field's value should be considered to be "ok" if not provided. It's not _exactly_ the same thing, and as such I agree with you on that account, but it's the only way of representing that idea in the OpenAPI spec as far as I know :)

Reply:
Indeed, the description contains the definition of the default value. Thus this addition is worth pulling in the new schemas for OPTIMADE.